src/prefect/utilities/tasks.py

```diff
@@ -45,13 +45,18 @@ def add(x, y):
         yield
 
 
-def as_task(x: Any, flow: Optional["Flow"] = None) -> "prefect.Task":
+def as_task(
+    x: Any, flow: Optional["Flow"] = None, convert_constants: bool = True
+) -> "prefect.Task":
     """
     Wraps a function, collection, or constant with the appropriate Task type.
 
     Args:
         - x (object): any Python object to convert to a prefect Task
         - flow (Flow, optional): Flow to which the prefect Task will be bound
+        - convert_constants (bool, optional): a boolean specifying whether all passed Python
```

Comment:
Your docstring points out this is slightly misnamed, since some "constants" (lists of constants) are never affected by this

Reply:
Yea this is fair; while we used to call "lists" constants, I think our previous terminology was the misnomer as those are more "collections of constants".  However, totally open to changing this if we can come up with a clearer name.